api/src/test/java/com/epam/pipeline/controller/WebTestConfiguration.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Could you please move all controller tests configuration into separate package `com.epam.pipeline.test.web`?